tox.ini

```diff
@@ -1,39 +1,47 @@
 [tox]
-envlist = py27,py36,static,pidiff
+envlist = py27,py36
```

Comment:
why not static?

Reply:
I doubt tox env static will run on python2.7 as it requires black which is only available for python3

Reply:
@rajulkumar, this I was not aware of. Thanks. I'll remove black.